scripts/test_templates.py

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i python3 -p python311Packages.requests
+
+import requests
+from time import sleep
+import sys
+
+TEMPLATE_TESTER_URL = "https://templatetester.tobyho.repl.co"
+HEADERS = {
+  "Authorization": "BLARGH",
```

Comment:
if this is a public repo we should create a semaphore secret for this

Reply:
Yup!